DataTool/ToolLogic/Extract/ExtractMusic.cs

```diff
@@ -49,7 +49,7 @@ public void ExtractType(ICLIFlags toolFlags)
             foreach (ulong @ulong in TrackedFiles[0x2C])
             {
                 STUSound music = GetInstance<STUSound>(@ulong);
-                var s_class = music.m_C32C2195.m_soundClass;
+                var s_class = music.m_C32C2195.m_wwiseBankID;
```

Comment:
also can we reformat this file? it doesn't follow the style guide of the project which is essentially braces on same line. The formatting rules are in the project, you should just be able to format the file? Did it not auto attempt to follow the project style? It does for me in Visual Studio unless it's related to Resharper?